### PR TITLE
Add media capture documentation to the README

### PR DESCRIPTION
**Pull request by nainkunal933**
**Issue #:**
Currently, we do not have any documentation on media pipelines in JS SDK readme. This can cause the customers to ignore this awesome feature.

**Description of changes:**
Adding links to blog post and developer documentation in the README

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
NA

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by devalevenkatesh — DISMISSED** [review]

**Comment by nainkunal933** [comment]
Will need input from other stakeholders, will merge after their approval

**Review by xuesichao — DISMISSED** [review]

**Review by achris — DISMISSED** [review]

**Review by schuettc — COMMENTED** [review]
There is a github repo that has been kept up to date more than the blog post.  Why not link directly to the github?  https://github.com/aws-samples/amazon-chime-media-capture-pipeline-demo

**Comment by nainkunal933** [comment]
> https://github.com/aws-samples/amazon-chime-media-capture-pipeline-demo

I will add this sample to the listed examples

File: README.md
Comment (nainkunal933) [inline README.md:120]: This is a very old example / blog post, it should be archived and removed

**Review by devalevenkatesh — APPROVED** [review]

**Pull request merged**
